testing/tests/DevExpress.ui.widgets.treeList/editing.tests.js

```diff
@@ -286,6 +286,62 @@ QUnit.test('AddRow method should expand row and add item after parent', function
     assert.strictEqual(rows[2].node.parent.key, 1, 'row 2 node parent');
 });
 
+QUnit.test('AddRow method should return Deferred with collapsed parent', function(assert) {
+    // arrange
+    this.options.editing.allowAdding = true;
+
+    this.setupTreeList();
+    this.rowsView.render($('#treeList'));
+    this.clock.tick();
+
+    // assert
+    assert.equal(this.getVisibleRows().length, 1, 'one visible row');
+
+    // act
+    this.addRow(1).done(() => {
+        // assert
+        assert.ok(true, 'addRow returns Deferred');
```

Comment:
If done is not called the test will be passed.

Reply:
Done